### PR TITLE
Reverse Geocoding

### PR DESCRIPTION
**Pull request by DejaTrudeaux**
connects to https://github.com/revelry-foundation/nightingale/issues/6

This PR includes configuration for google places api (hiding api keys in rn) as well as reverse geocoding to make coordinates human readable

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,44 +1,45 @@
...
+        - git clean -d -f
+        - gem install bundler:2.0.2
+        - bundle install
+      before_script:
+        # set up SSH for pulling the private signing certs
+        - openssl aes-256-cbc -K $encrypted_db4416ddba70_key -iv $encrypted_db4416ddba70_iv -in ./ci/certs_key.enc -out ./ci/certs_key -d
+        - chmod 600 ./ci/certs_key
+        - eval "$(ssh-agent -s)"
+        - ssh-add ./ci/certs_key
+        - echo "GOOGLE_PLACES_API_KEY=$GOOGLE_PLACES_API_KEY" > .env
```
Comment (DejaTrudeaux) [inline .travis.yml:37]: should be the only difference here

File: src/containers/LocationStorageContainer.tsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import {Container} from 'unstated'
 import SInfo from 'react-native-sensitive-info'
+import {formatAddressInfo} from '../navigation/CoordinateTranslator'
 
 export interface Location {
   latitude: number
   longitude: number
+  address: string
```
Comment (DejaTrudeaux) [inline src/containers/LocationStorageContainer.tsx:8]: reverse geocoded location

File: .travis.yml [hunk 3]
```diff
@@ -34,6 +34,7 @@ jobs:
     - chmod 600 ./ci/certs_key
     - eval "$(ssh-agent -s)"
     - ssh-add ./ci/certs_key
+    - echo "GOOGLE_PLACES_API_KEY=$GOOGLE_PLACES_API_KEY" > .env
```
Comment (CurtisVonRubenhoff) [inline .travis.yml:37]: you might need to add this in above for the testing step

Comment (DejaTrudeaux) [reply .travis.yml:38]: i did thank u!

**Pull request merged**